experimental/formatted_string/src/formatted_string.rs

```diff
@@ -64,14 +64,28 @@ impl<F: Copy, const L: usize> FormattedStringLike<F, L> for LayeredFormattedStri
 
 impl<F: Copy, const L: usize> LayeredFormattedString<F, L> {
     pub fn new() -> Self {
+        Self::with_capacity(40)
```

Comment:
I can imagine someone wanting to create an empty FormattedString and never mutate it.  In that case, I'd probably err on the side of having the `new()` function return an empty string with an empty vec, simply because then we don't needlessly allocate memory.

Reply:
@robertbastian yeah vec guarantees this

Reply:
Let's just inherit the underlying collection's behaviour.